deploy/role.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```

Comment:
maybe worth to remove file name to `cluster_role.yaml`, it looks confusing to me

Reply:
done in 930cb5a

Reply:
@corinnekrych @dipak-pawar 
With this change, we cannot add a new API because operator sdk complains of the missing `role.yaml` file
```
➜  devopsconsole-operator git:(devopsconsole-installer) ✗ operator-sdk add api --api-version=devopsconsole.openshift.io/v1alpha1 --kind=Installer            
INFO[0000] Generating api version devopsconsole.openshift.io/v1alpha1 for kind Installer. 
INFO[0000] Created pkg/apis/devopsconsole/v1alpha1/installer_types.go 
INFO[0000] Created pkg/apis/addtoscheme_devopsconsole_v1alpha1.go 
INFO[0000] Created deploy/crds/devopsconsole_v1alpha1_installer_cr.yaml 
INFO[0001] Created deploy/crds/devopsconsole_v1alpha1_installer_crd.yaml 
Error: failed to update the RBAC manifest for the resource (devopsconsole.openshift.io/v1alpha1, Installer): (failed to read role manifest /home/ijarif/Projects/go/src/github.com/redhat-developer/devopsconsole-operator/deploy/role.yaml: open /home/ijarif/Projects/go/src/github.com/redhat-developer/devopsconsole-operator/deploy/role.yaml: no such file or directory)
Usage:
  operator-sdk add api [flags]

Flags:
      --api-version string   Kubernetes APIVersion that has a format of $GROUP_NAME/$VERSION (e.g app.example.com/v1alpha1)
  -h, --help                 help for api
      --kind string          Kubernetes resource Kind name. (e.g AppService)

```
The operator-sdk looks for `role.yml` file. See https://github.com/operator-framework/operator-sdk/blob/master/commands/operator-sdk/cmd/add/api.go#L119 and https://github.com/operator-framework/operator-sdk/blob/master/pkg/scaffold/role.go#L33

I would like to rename the `cluster_role.yaml` to `role.yaml` and I've raised https://github.com/redhat-developer/devopsconsole-operator/pull/30 for it. @corinnekrych can you please take a look at that PR?